bin/install.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+set -eu
+
+# Ensure PHP is installed
+if ! command -v php > /dev/null 2>&1; then
+  echo "PHP is not found, installation aborted!"
+  exit 1
+fi
+
+# Ensure Composer is installed/updated
+if ! command -v composer > /dev/null 2>&1; then
+  echo "*** Installing Composer ***"
```

Comment:
Yeah... Since we don't have a convention for it yet... Nevertheless, nice link, will "borrow" some. 😉